apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -66,6 +66,10 @@ type ContourDeploymentSpec struct {
 	//
 	// +optional
 	RuntimeSettings *ContourConfigurationSpec `json:"runtimeSettings,omitempty"`
+
+	// CommonLabels is a set of labels to add to the provisioned Contour resources.
+	// +optional
+	CommonLabels map[string]string `json:"commonLabels,omitempty"`
```

Comment:
I'm thinking maybe `ResourceLabels` would be a slightly more descriptive name, @sunjayBhatia WDYT?

Reply:
yep, i like that name 👍🏽

Reply:
done